### PR TITLE
Olivekl docs cleanup 3

### PR DESCRIPTION
**Pull request by olivekl**
General edits to running_clusterfuzzlite.md and github_actions.md for clarity and concision

### CODE REVIEW

File: docs/running-clusterfuzzlite/github_actions.md [hunk 1]
```diff
@@ -90,35 +93,29 @@ jobs:
...
 [later on]: #storage-repo
 
 ### Batch fuzzing
 
-Batch fuzzing enables continuous, regular fuzzing on your latest HEAD, and
-allows a corpus of inputs to build up over time that greatly improves the
-effectiveness of fuzzing.
+Batch fuzzing enables continuous, regular fuzzing on your latest HEAD and
+allows a corpus of inputs to build up over time, which greatly improves the
+effectiveness of fuzzing. Batch fuzzing can be run on either pushes to your default branch or on a cron schedule (or both).
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:114]: Take out the part about pushes please. I don't want to encourage this.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 2]
```diff
@@ -159,37 +156,35 @@ jobs:
...
 
-Continuous builds allow code change fuzzing to detect if crashes were
-introduced by the change under test.
-If the crash is not novel, code change/PR fuzzing will not report it.
+The continuous build task causes a build to be triggered and uploaded as a GitHub Actions artifact
+whenever a new push is done to main/default branch.
 
-Continuous builds are used whenever a crash is found during PR fuzzing to determine if this crash was newly introduced.
+Continuous builds are used when a crash is found during PR fuzzing to determine whether the crash was newly introduced.
+If the crash is not novel, code change/PR fuzzing will not report it.
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:187]: Maybe drop "code change/"

File: docs/running-clusterfuzzlite/github_actions.md [hunk 3]
```diff
@@ -308,15 +295,13 @@ jobs:
 ```
 {% endraw %}
 
-You can copy and paste the above file and pruning should just work.
-But here is where `storage-repo`, `storage-repo-branch`, and
-`storage-repo-branch-coverage` come in handy.
+Optionally, edit the following fields to view coverage reports at
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:298]: Hmmm... do we tell users they should change OWNER and STORAGE-REPO-NAME to the appropriate values for their storage repo?

File: docs/running-clusterfuzzlite/github_actions.md [hunk 4]
```diff
@@ -343,19 +328,19 @@ storage repo and add it as a [repository secret] called
...
 ![new-repo-secret]
 ![personal-access-token]
 
-If you would like PR fuzzing to only run fuzzers affected by the current
+If you would like PR fuzzing to run only the fuzzers affected by the current
 change, you'll need to add these same options to the ["Build Fuzzers" step
-above](#pr-fuzzing). The way "affected fuzzers" are determined is by using
-coverage reports.
+above](#pr-fuzzing). The "affected fuzzers" are determined by using
+coverage reports, which require a storage repo.
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:340]: It doesn't require this.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: docs/running-clusterfuzzlite/github_actions.md (hunk 3 above)
Comment (olivekl) [reply docs/running-clusterfuzzlite/github_actions.md:298]: I've added a third line: 

-  set `storage-repo` ([instructions here])
- set `storage-repo-branch-coverage` to "gh-pages" (the
default)
- set `owner` and `storage-repo-name` to the appropriate values for your storage repo

Does the first line (set `storage-repo`) still make sense now? Maybe it should be "set up a storage repo" with the link to the #storage-repo section?

Comment (jonathanmetzman) [reply docs/running-clusterfuzzlite/github_actions.md:298]: What you have makes sense. But I wonder if it should be in a more general place. Don't worry I can fix this.

**Pull request merged**